pages/index.js

```diff
@@ -101,7 +100,11 @@ export default function HomePage() {
           console.log("history: ", history);
         }}
       >
-        history test button
+        simuliere:
+        <br />
+        Gestern 23:59 {`==>`} Heute 00:00 mit heutigen Werten.
+        <br />
+        Zum erneuten Testen bitte local storage löschen.
```

Comment:
Ok, but I still don't see how this reflects your user story: there is no such a button. And what value would this button have for the user?